api/api.proto

```diff
@@ -2058,10 +2058,14 @@ message SdkCloudBackupCreateRequest {
   string credential_id = 2;
   // Full indicates if full backup is desired even though incremental is possible
   bool full = 3;
+  // Name of the task performing this backup
```

Comment:
Probably need to add `...this backup. This value is required and used for idempotency.`

Reply:
ok.